docs/datagatherers/aks.md

```diff
@@ -0,0 +1,36 @@
+# AKS Data Gatherer
+
+The AKS *data gatherer* fetches information about a cluster from the Azure
+Kubernetes Service API.
+
+## Data
```

Comment:
We also have an example of the data collected here: https://github.com/jetstack/preflight/tree/master/examples/data

Reply:
Hmm, forgot about those. My preference would be, for now, to keep all this information in one place - or at least local to the data gatherer folder (and not some other dir in the project).

Main reason being I'm keen to keep the number of places changes to dg and other config down.

If it were up to me, I'd delete that examples/data folder since we have links to the data here in the readmes now.

Reply:
Removed the examples in 527ff171fef87e3f4cfe8312ea3a2dc4977abe82 - reasoning repeated in the commit message.